services/appflowy-collaborate/src/collab/storage.rs

```diff
@@ -75,6 +89,103 @@ where
     }
   }
 
+  const PENDING_WRITE_BUF_CAPACITY: usize = 100;
+  async fn periodic_write_task(cache: CollabCache, mut reader: Receiver<PendingCollabWrite>) {
+    let mut buf = Vec::with_capacity(Self::PENDING_WRITE_BUF_CAPACITY);
```

Comment:
Is it possible for Self::persist to be called very frequently?

Reply:
According to documentation, `reader.recv_many` used in a loop below will await until at least one element has been passed to a reader or the sender channel has been closed.